src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+    firstPromise();
+    resetTimer();
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
+
+  if (isClickedLeft !== isClickedRight) {
+    secondPromise();
+  }
+
+  if (isClickedLeft && isClickedRight) {
+    thirdPromise();
+  }
+});
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+async function firstPromise() {
+  return new Promise((resolve, reject) => {
+    if (isClickedLeft) {
+      resolve('First promise was resolved');
+    } else {
+      reject(new Error('First promise was rejected'));
+    }
+  })
+    .then((result) => {
+      notification.classList.add('success');
+      notification.classList.remove('error');
+      notification.innerText = result;
+      document.body.prepend(notification);
+    })
+    .catch((error) => {
+      notification.classList.add('error');
```

Comment:
The same issue as in line 52. You're adding the 'error' class every time a promise is rejected. This could lead to unnecessary DOM manipulation if the 'error' class is already present.